src/test/java/com/spotify/docker/client/DockerHostTest.java

```diff
@@ -98,7 +98,7 @@ public void testCertPathFromEnv() throws Exception {
     DockerHost.setSystemDelegate(systemDelegate);
 
     assertThat(DockerHost.certPathFromEnv(), equalTo("foo"));
-    assertThat(DockerHost.certPathFromEnv(), equalTo("bar/.docker"));
+    assertThat(DockerHost.certPathFromEnv(), equalTo(null));
```

Comment:
how can the same method evaluate to `"foo"` and then `null`? What is the point of returning different values from the mock?

you can also use `nullValue()` instead of `equalTo(null)`